cars/v1/x_pack/monitoring/templates/config/elasticsearch.yml

```diff
@@ -2,8 +2,8 @@ xpack.monitoring.collection.enabled: true
 
 xpack.monitoring.exporters.custom:
   type: {{monitoring_type}}
-{%- if monitoring_host is defined and monitoring_port is defined %}
-  host: ["{{monitoring_host}}:{{monitoring_port}}"]
+{%- if monitoring_host is defined %}
+  host: ["{{monitoring_scheme|default('http')}}://{{monitoring_host}}:{{monitoring_port|default('9200')}}"]
```

Comment:
```suggestion
  host: ["{{monitoring_scheme|default('http')}}://{{monitoring_host}}:{{monitoring_port|default('9200')}}"]
{%- elif monitoring_host is defined %}
  host: ["{{monitoring_host}}:{{monitoring_port|default('9200')}}"]
```

Reply:
Currently, it's not possible to work around by explicitly adding `https://`, this is the main reason I'm opening this pull request :)

```
$ esrally race --distribution-version=7.12.0 --track=geonames --car="4gheap,x-pack-monitoring-http" --client-options="timeout:60" --car-params="monitoring_type:'http',monitoring_host:'https://HOST',monitoring_port:9243,monitoring_user:'elastic',monitoring_password:'SECRET'"

    ____        ____
   / __ \____ _/ / /_  __
  / /_/ / __ `/ / / / / /
 / _, _/ /_/ / / / /_/ /
/_/ |_|\__,_/_/_/\__, /
                /____/

[ERROR] Cannot race. too many values to unpack (expected 2).
```

Anyway, I'm happy to adopt your suggestions.

Reply:
If you believe it's more appropriate to just use the protocol with the host name, you can accomplish this by pointing the `--car-params` flag to a json file with the values instead of using inline key-value pairs, e.g. `--car-params=car-params.json`

car-params.json:
```json
{
  'monitoring_type': 'http',
  'monitoring_host': 'https://HOST',
  'monitoring_port': 9243,
  'monitoring_user': 'elastic',
  'monitoring_password': 'SECRET'
}
```

Reply:
Interesting, thank you for pointing that out!